package.json

```diff
@@ -40,6 +40,7 @@
     "zod": "^3.22.4"
   },
   "devDependencies": {
+    "@hookform/devtools": "^4.3.1",
```

Comment:
**React Hook Form DevTools** are extremely useful for debugging forms built on [react-hook-form](https://www.react-hook-form.com/).
#### Main features:
- debugger in the sidebar on top of the browser page, which does not affect the page layout
- Live state values of the forms fields, such as `isDirty`, `isTouched`, `value`, etc
- Live values of the entire form state, such as `isSubmitting`, `isValid`, `isDirty`, etc

Her's the page with the [Live Demo](https://www.react-hook-form.com/dev-tools/)